internal/tools/orchestrator/scheduler/executor/plugins/k8s/k8s.go

```diff
@@ -458,9 +447,9 @@ func New(name executortypes.Name, clusterName string, options map[string]string)
 	k.elasticsearchoperator = elasticsearchoperator
 	redisoperator := redis.New(k, deploy, sts, svc, ns, k, k8ssecret, client)
 	k.redisoperator = redisoperator
-	mysqloperator := mysql.New(k, ns, k8ssecret, pvc, client)
+	mysqloperator := mysql.New(k, ns, k, k8ssecret, pvc, client)
```

Comment:
two `k` is strange.

Reply:
Optimized, create new module over subscribe ratio.